src/args.jl

```diff
@@ -532,6 +533,7 @@ add_aliases(:gridlinewidth, :gridwidth, :grid_linewidth, :grid_width, :gridlw, :
 add_aliases(:gridstyle, :grid_style, :gridlinestyle, :grid_linestyle, :grid_ls, :gridls)
 add_aliases(:framestyle, :frame_style, :frame, :axesstyle, :axes_style, :boxstyle, :box_style, :box, :borderstyle, :border_style, :border)
 add_aliases(:tick_direction, :tickdirection, :tick_dir, :tickdir, :tick_orientation, :tickorientation, :tick_or, :tickor)
+add_aliases(:camera, :cam)
```

Comment:
How about `:view`, `:viewangle` and `:view_angle` as additional aliases?

Reply:
I wanted to add `:view` (and others) but in the original thread tbreloff was against it. If we currently feel those are good I can add them 😄

Reply:
I think if they are not conflicting with other attribute names or aliases it's fin to have them as aliases.